frontend/src/components/gotEnzymes/Landing.vue

```diff
@@ -0,0 +1,185 @@
+<template>
+  <div>
+    <section class="hero is-primary is-bold py-6">
+      <div class="hero-body has-text-centered">
+        <p class="is-size-1 title">GotEnzymes</p>
+        <p class="is-size-5">We've got the enzymes you need</p>
+      </div>
+    </section>
+    <div class="container is-fullhd">
+      <div class="columns is-centered pt-6">
+        <div
+          class="column is-three-fifths-desktop is-three-quarters-tablet is-fullwidth-mobile control"
+        >
+          <div>
+            <p class="control has-icons-right has-icons-left">
+              <input
+                id="search"
+                v-model="searchTerm"
+                data-hj-whitelist
+                class="input"
+                type="text"
+                placeholder="search"
+                @keyup.enter="updateSearch()"
+                @input="searchStringChange()"
+              />
+              <span
+                v-show="showSearchCharAlert"
+                class="has-text-info icon is-right"
+                style="width: 220px"
+              >
+                Type at least 2 characters
+              </span>
+              <span class="icon is-medium is-left">
+                <i class="fa fa-search is-primary"></i>
+              </span>
+            </p>
+            <div v-if="searchResultsEmpty" class="mt-2">
```

Comment:
Is this condition here really needed? I see this variable is used, but I'm not sure it should affect the visibility of anything.

Reply:
Good catch! The `div` nesting was unnecessary and the variable name was misleading. Addressed in https://github.com/MetabolicAtlas/MetabolicAtlas/pull/1031/commits/268f2fbd3f0684d5496aaedbf3ac4123690d3e9f